TransformationSystem/Agent/TransformationCleaningAgent.py

```diff
@@ -427,10 +379,8 @@ def removeTransformationOutput(self, transID):
         res = self.cleanCatalogContents(directory)
         if not res['OK']:
           return res
-        res = self.cleanStorageContents(directory)
-        if not res['OK']:
-          return res
-    self.log.info("Removed directories in the catalog and storage for transformation")
+
+    self.log.info("Removed directories in the catalog for transformation")
```

Comment:
That is not quite true. From what I see, the `cleanCatalogContents` calls the DataManager class, so it acts both on the catalog and the storage, but uses the catalog as a starting point

Reply:
Right. This is probably the correct thing to do, but the name of the method, and the log, are wrong. Correct?

Reply:
i'd say so yes